guides/v2.2/config-guide/mq/manage-mysql.md

```diff
@@ -66,7 +66,7 @@ Edit */app/etc/env.php* file for configure cron job `consumers_runner`
 
 Use the `magento` command to start message queue consumers. You can start multiple consumers simultaneously.
 
-    ./bin/magento queue:consumers:start [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] <consumer_name>
+    ./bin/magento queue:consumers:start <consumer_name> [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] [--area-code=<value>] 
```

Comment:
Does this not also apply to 2.1 and 2.3?

Reply:
I have updated for v2.1. Check it.